### PR TITLE
Kpi/grafana dashboard

### PR DESCRIPTION
**Pull request by soerenbf**
## Purpose

Importable configuration of grafana dashboard. A readme has also been created, detailing configuration of grafana.

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.

### CODE REVIEW

File: kpi-tracker/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+This folder contains the source code for the KPI-tracker service used to collect relevant metrics from the chain.
+
+## Grafana
+
+We use Grafana to visualize the data collected by the service. The configuration of the dashboard can be found under [grafana/dashboard.json](./grafana/dashboard.json).
+
+### Installation and running
+
+Follow the steps to [install grafana](https://grafana.com/docs/grafana/latest/setup-grafana/installation/) for your OS, which should also the steps required to run the installed service.
```
Comment (abizjak) [inline kpi-tracker/README.md:11]: "should also the steps"

File: kpi-tracker/grafana/dashboard.json [hunk 2]
```diff
@@ -0,0 +1,1326 @@
...
+      ],
+      "title": "Active accounts (historical)",
+      "type": "timeseries"
+    },
+    {
+      "datasource": {
+        "type": "postgres",
+        "uid": "${postgres_db}"
+      },
+      "description": "This will be slow, as for every date we try to look up all transactions for the past 30 days",
```
Comment (abizjak) [inline kpi-tracker/grafana/dashboard.json:304]: Is that still an accurate description here? Why is 30 hardcoded?

**Review by abizjak — APPROVED** [review]
I haven't had time to try it, but looks OK.

File: kpi-tracker/grafana/dashboard.json (hunk 2 above)
Comment (soerenbf) [reply kpi-tracker/grafana/dashboard.json:304]: good catch, will remove

**Pull request merged**